.github/workflows/tag.yml

```diff
@@ -0,0 +1,61 @@
+name: CI to Docker Hub
```

Comment:
Can you use this tag workflow as a starting point instead https://github.com/blendededge/filter-component-oih/blob/master/.github/workflows/tag.yml#L7?  It it based on the released event from Github instead of the tag.